tensorflow_datasets/core/image_visualizer.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Lint as: python3
+
+"""Visualizer for image datasets.
+"""
+
+from absl import logging
+
+from tensorflow_datasets.core import dataset_utils
+from tensorflow_datasets.core import features as features_lib
+from tensorflow_datasets.core import lazy_imports_lib
+
+
+def match(ds_info):
+  """Checks if dataset belongs to image dataset category"""
+  image_keys = [
+      k for k, feature in ds_info.features.items()
+      if isinstance(feature, features_lib.Image)
+  ]
```

Comment:
When you are re-using the same pattern multiple times, it's good to make a function and reuse it every-where. For instance:

`image_keys = _extract_keys(ds_info, features_lib.Image)`